### PR TITLE
refs #13413 - removing ability for docker content upload

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

File: app/controllers/katello/api/v2/repositories_controller.rb [hunk 1]
```diff
@@ -249,6 +249,8 @@ def remove_content
     def upload_content
       filepaths = Array.wrap(params[:content]).compact.map(&:path)
 
+      fail Katello::Errors::InvalidRepositoryContent, "Cannot upload Docker content." if @repository.docker?
```
Comment (daviddavis) [inline app/controllers/katello/api/v2/repositories_controller.rb]: Should this be translated?

Comment (komidore64) [inline app/controllers/katello/api/v2/repositories_controller.rb]: Hah, you're right! Needs to be translated, thank you.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html [hunk 2]
```diff
@@ -260,7 +260,6 @@ <h4 translate>Content Counts</h4>
   <section class="well" ng-if="permitted('edit_products', product) && !product.redhat">
     <h5 translate ng-show="repository.content_type == 'puppet'">Upload Puppet Module</h5>
     <h5 translate ng-show="repository.content_type == 'yum'">Upload Package</h5>
-    <h5 translate ng-show="repository.content_type == 'docker'">Upload Docker Image</h5>
```
Comment (komidore64) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html]: Spoke with @jlsherrill after PR'ing this, I need to adding another conditional to the `ng-if` statement to not show the entire section.

**Comment by komidore64** [comment]
@daviddavis @jlsherrill updated

**Comment by komidore64** [comment]
labels are hard

**Comment by daviddavis** [comment]
ACK from me.

**Comment by jlsherrill** [comment]
maybe reopen against master?

**Comment by komidore64** [comment]
ah yes. right

**Comment by komidore64** [comment]
closing to reopen against master.

**Pull request closed without merging**